rust/basic_bitcoin/src/basic_bitcoin/src/bitcoin_api.rs

```diff
@@ -0,0 +1,91 @@
+use ic_btc_types::{
+    GetBalanceRequest, GetCurrentFeePercentilesRequest, GetUtxosRequest, GetUtxosResponse,
+    MillisatoshiPerByte, Network, SendTransactionRequest,
+};
+use ic_cdk::{api::call::call_with_payment, export::Principal};
+
+// The fees for the various bitcoin endpoints.
```

Comment:
Why are some called `_COST_CYCLES` and others `_FEE`?
I like the first option because it contains more information, and there is no risk that somebody confuses it for a Bitcoin fee.

Reply:
Agreed, copy pasta. Updated to use `_CYCLES` everywhere.